cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/bulkwriter/token/TokenRangeMappingTest.java

```diff
@@ -105,16 +101,145 @@ void testCreateTokenRangeMappingWithPending()
         }
         TokenRangeMapping<RingInstance> topology = TokenRangeMapping.create(() -> response,
                                                                             () -> Partitioner.Murmur3Partitioner,
-                                                                            () -> rf, RingInstance::new);
+                                                                            RingInstance::new);
         assertThat(topology.pendingInstances()).hasSize(pendingCount);
     }
 
+    @Test
+    void testConsolidateListOfTokenRangeMappingFails()
```

Comment:
Is it possible to add some tests (or modify the replica building logic) to cover the entire token range of the partitioner so we have wrap-around ranges in these tests please?

Reply:
The `TokenRangeMapping` covers the entire ring and contains only the unwrapped ranges already.